home.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="header">
+      <div class="header-frame">
+        <a href="/" class="title-box">
+          <img src="./images/판다 얼굴.png" class="logo">
```

Comment:
```suggestion
          <img src="./images/판다 얼굴.png" class="logo" alt="">
```
파일 이름에 공백, 한글은 권장하지 않습니다.

영문으로, 그리고 공백이 필요하다면 `-` 하이픈이나 `_` 언더스코어로 문자열이 끊기지않도록 해주세요.

img에 alt 속성이 없는데, 어떤 속성인지 찾아보시면 좋을것같습니다 :-0